.github/labeler.yml

```diff
@@ -3,6 +3,12 @@ documentation:
 - LICENSE*
 - .github/FUNDING.yml
 
+announcements:
+- src/components/AnnouncementsPage/**/*
+- src/components/HomePage/HomeAnnouncementBanner.js
+- src/data/announcements.json
+- src/pages/announcements.js
+
```

Comment:
I think there's another file that has to do with announcements: update_announcements.py
Tho idk if this file is only for .js or .json files?
(side note: I finally spelled announcements right the first time ya yeet)